style.md

```diff
@@ -13,9 +13,12 @@ The redundancy reduction from removing the namespacing prefix is not useful enou
 ## Optional settings should have pointer Go types
 
 So we have a consistent way to identify unset values ([source][optional-pointer]).
+The exceptions are slices, maps, and booleans where the default should be false, in which case `omitempty` is sufficient and no pointer is needed ([source][no-pointer-for-slices] and [source][no-pointer-for-boolean]).
```

Comment:
On Tue, Jan 26, 2016 at 06:03:44PM -0800, Qiang Huang wrote:
> > But sometimes false does mean “don't change” (and those are the
> > booleans where the default should be false)
> 
> That'll be a problem if we want some `bool` values to be updated
> (live update), we surely want to change some booleans from `true` to
> `false`.

Yeah, booleans where ‘false’ requires action should use pointers.  I
guess the wording in 60b83df is based on my assumption that “default”
equals “don't do anything” [1].  I've pushed 60b83df → d715acf with
wording to explicitly pick out “Go's default for the type is a no-op
in the spec” cases.

[1]: https://github.com/opencontainers/specs/pull/285#issuecomment-167823651